src/lib/hooks/useNetworkFees.ts

```diff
@@ -0,0 +1,198 @@
+import { Coins, Services } from '@ardenthq/sdk';
```

Comment:
general remark for this file, have a look at what's really necessary from here for arkconnect and strip out the rest to avoid obsolete code